samples/boards/espressif/ulp/lp_core/interrupt_ulp/src/main.c

```diff
@@ -0,0 +1,20 @@
+/*
+ * Copyright (c) 2025 Espressif Systems (Shanghai) Co., Ltd.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <stdio.h>
+#include <ulp_lp_core.h>
+#include <zephyr/kernel.h>
+
+int main(void)
+{
+	while (1) {
+		printf("Triggering ULP interrupt...\n");
```

Comment:
Since the memory map from lp core is accessible from hp core why not also print the interrupt vcounts from the HP core? It would add more value to the sample.

Reply:
Totally, but that would require some shared memory description and ipc mechanism which is not quite ready for LP Core. The idea here was to showcase just a simple interrupt triggered by HP core. Some more nice features are coming soon, such as ipc and waking HP core from LP core.